parquet_file/src/catalog.rs

```diff
@@ -164,6 +166,12 @@ pub enum Error {
 
     #[snafu(display("Catalog already exists"))]
     AlreadyExists {},
+
+    #[snafu(display("Datetime required but missing"))]
```

Comment:
```suggestion
    #[snafu(display("Internal: Datetime required but missing in serialized catalog"))]
```
I wonder if that is a more specific error